### PR TITLE
Feature #154 - Tests for put patch method for editing business info

### LINKED ISSUE #154: SP: 3 Tests for PUT/PATCH method for editing business info

### PR DESCRIPTION
**Pull request by therobotisnotatoy**

### CODE REVIEW

File: beauty/api/tests/test_business_views.py [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+        self.groups.owner.user_set.add(self.owner)
+        self.groups.owner.user_set.add(self.another_user)
+        self.business = BusinessFactory.create(owner=self.owner)
+
+    def test_get_method_for_specific_owner_businesses_view_by_unauthenticated_user(self):
+        """Unauthenticated user can not view list of his/her all businesses."""
+        self.client.force_authenticate(user=None)
+        response = self.client.get(reverse(
+            viewname="api:businesses-list-create",
+            kwargs={}),
```
Comment (pazuzu-ua) [inline beauty/api/tests/test_business_views.py:50]: Is there need to pass an empty dict to kwargs? It should work without it.

File: beauty/api/tests/test_business_views.py [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+            kwargs={
+                "pk": self.business.id,
+            }),
+        )
+        self.assertEqual(response.status_code, 200)
+        self.assertIs(response.data.get("id"), None)
+
+    def test_get_method_for_edit_view_by_real_owner_of_business(self):
+        """Owner of the business can access its business edit view."""
+        self.client.force_authenticate(user=self.business.owner)
```
Comment (pazuzu-ua) [inline beauty/api/tests/test_business_views.py:139]: If you have so many tests with 
`self.client.force_authenticate(user=self.business.owner)`
you can authenticate user once in setUp, and where you need any other logic (like other authentication or no user), you can do it specifically

**Review by pazuzu-ua — APPROVED** [review]

File: beauty/api/tests/test_business_views.py (hunk 1 above)
Comment (therobotisnotatoy) [reply beauty/api/tests/test_business_views.py:50]: It doesn't. The day before I used URL with additional "arguments" And I decided to leave it as it is.

File: beauty/api/tests/test_business_views.py (hunk 2 above)
Comment (therobotisnotatoy) [reply beauty/api/tests/test_business_views.py:139]: Yeap =)

**Review by Misha86 — APPROVED** [review]

**Pull request merged**